consensus/rules/src/batch_verifier.rs

```diff
@@ -0,0 +1,29 @@
+use multiexp::BatchVerifier as InternalBatchVerifier;
+
+/// This trait represents a batch verifier.
+///
+/// A batch verifier is used to speed up verification by verifying multiple transactions together.
+///
+/// Not all proofs can be batched and at it's core it's intended to verify a series of statements are
```

Comment:
I... can't... help it ahhhhhhhhhhh

it's = it is
its = possession

I'm sorry in advance.

```suggestion
/// Not all proofs can be batched and at its core it's intended to verify a series of statements are
```